ruby/rails/config.reek

```diff
@@ -0,0 +1,25 @@
+---
+
+Attribute:
+  enabled: true
```

Comment:
I was merging the ruby/config.reek file into this one. Does this line not play well with `InstanceVariableAssumption` below, which does fix my ivar reek problem?